src/main/java/org/onebusaway/alexa/AnonSpeechlet.java

```diff
@@ -261,4 +266,23 @@ private SpeechletResponse askForCity(Optional<String> currentCityName) {
         }
         return SpeechletResponse.newAskResponse(out, cityReprompt);
     }
+
+    private String allRegionsSpoken() {
+        List<String> activeRegions = obaClient.getAllRegions()
+                .stream()
+                .filter(r -> RegionUtils.isRegionUsable(r) && !r.getExperimental())
```

Comment:
oba-client-library updated via https://github.com/OneBusAway/onebusaway-client-library/pull/10.  New SNAPSHOT should be available via OBA Maven repo.